.github/workflows/ci.yml

```diff
@@ -42,7 +42,11 @@ jobs:
       run: cargo test --locked --verbose
 
   end-to-end-tests:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
I think we need to add `shell: bash` to the "run" step on line 55, otherwise we run into issues with trying to run a `.sh` that causes the end-to-end failures here